### DISCUSSION TITLE
REQ] Please auto generate a linked CSS file

### DISCUSSION THREAD

**Discussion by hoshi411** [senshu/Sozi#615] [Ideas]
I know that I can manually create a CSS file and add it to the generated html file.
But then every time Sozi generates a new html file the CSS link gets deleted.
Please auto generate an attatched CSS file so that I can use the CSS file easily.

I also recommend adding to this boilerplate attatched CSS file a preformatted link to the fonts used in the document which would allow the developer to simply copy the font files into the same local folder as the SVG if they wanted to.

**Comment by hoshi411** [comment]
I recommend using the following CSS file which I developed and works perfectly to drop a font file into your directory .
It is tedious to have to link this css every single time the html file gets generated.

https://www.codepile.net/pile/W9jJgJ9R

**Comment by aumouvantsillage** [comment]
> But then every time Sozi generates a new html file the CSS link gets deleted.

I'm sorry but I don't understand what you mean.
Which CSS link are you referring to?

When you use the feature "Custom stylesheets and scripts" in the presentation editor, Sozi will copy your stylesheet into the generated HTML. It will not create or remove a CSS link.

**Reply by hoshi411** [reply]
I didn't realize that Sozi had the feature to add CSS from a file.

My ideal would be that Sozi automatically generate an external CSS file which is linked to the HTML file automatically.

It is good news to me that an external CSS file can be "copied into" by checking a preference.  I will look into that.

If that works then that is a solution, though backwards from my ideal.

**Comment by hoshi411** [comment]
I just opened sozi and realized that this feature already exists.  I missed it.  I can simply scroll down a little and add a css file or js file.  This is awesome.  Thank you for being patient with me.
